### PR TITLE
Frontend develoment milestone 5th

### PR DESCRIPTION
**Pull request by kshekh**
Include the issue number you are working on

[ ] Bug Fix
[ ] Refactor
[ ] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description
Removed cookies code and added local storage instead

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Log in as '...'
2. Go to '...' 
3. Click on '...'
4. See the updated element x

**Preview PMU form after Building**
http://localhost:5173/user-view-form
http://localhost:5173/settings-index

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

File: src/router/index.js.bak [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import { createWebHistory, createRouter } from 'vue-router';
```
Comment (giteshk11) [inline src/router/index.js.bak:1]: Why is this file here?

**Review by giteshk11 — CHANGES_REQUESTED** [review]

File: src/router/index.js.bak (hunk 1 above)
Comment (kshekh) [reply src/router/index.js.bak]: This has been removed. Please check again

**Comment by paulchrisluke** [comment]
Screenshot 2023-01-29 at 11 52 07 AM
Screenshot 2023-01-29 at 11 53 58 AM


3px miss alignment of media and forms icons. They are 3px to the right of the rest of the icons, they all should be in the same line.

**Review by giteshk11 — APPROVED** [review]
